brand-guidelines/branded-tools/README.md

```diff
@@ -0,0 +1,255 @@
+![All branded tools logos](./assets/branded-tools-all-logos.png)
+
+# Tooling logos documentation
+
+Please refer to the following documentation for proper usage of the logos created for AsyncAPI's branded open source tools.
+
+**Table of contents**
+- Studio
+    - [Studio logo (light)](#studio-logo-light)
+    - [Studio logo (dark)](#studio-logo-dark)
+    - [Download logo kit for Studio](#download-logo-kit-for-studio)
+- Modelina
+    - [Modelina logo (light)](#modelina-logo-light)
+    - [Modelina logo (dark)](#modelina-logo-dark)
+    - [Download logo kit for Modelina](#download-logo-kit-for-modelina)
+- Glee
+    - [Glee logo (light)](#glee-logo-light)
+    - [Glee logo (dark)](#glee-logo-dark)
+    - [Download logo kit for Glee](#download-logo-kit-for-glee)
+- Parser
+    - [Parser logo (light)](#parser-logo-light)
+    - [Parser logo (dark)](#parser-logo-dark)
+    - [Download logo kit for Parser](#download-logo-kit-for-parser)
+- CLI
+    - [CLI logo (light)](#cli-logo-light)
+    - [CLI logo (dark)](#cli-logo-dark)
+    - [Download logo kit for CLI](#download-logo-kit-for-cli)
+- Github Actions
+    - [Github Actions logo (light)](#github-actions-logo-light)
+    - [Github Actions logo (dark)](#github-actions-logo-dark)
+    - [Download logo kit for Github Actions](#download-logo-kit-for-github-actions)
+- Generator
+    - [Generator logo (light)](#generator-logo-light)
+    - [Generator logo (dark)](#generator-logo-dark)
+    - [Download logo kit for Generator](#download-logo-kit-for-generator)
+
+---
+
+## Studio logo (light)
+
+This version of the logo should be used on dark background interfaces.
+
+![Studio logo (light)](./assets/tooling-studio-light.png)
+
+| Spec | Details |
+| --- | --- |
+| Primary color | Studio Pink - `#E50E99` |
+| Type color | [`white`](../color/README.md/#base-colors) |
+| Background color | [`dark`](../color/README.md/#base-colors) |
+
+
+## Studio logo (dark)
```

Comment:
I see you're doing headings like Studio logo (light), Studio logo (dark), Modelina logo (light), Modelina logo (dark)...

What about doing it like this so it doesn't look so repetitive?

```markdown
## Studio logo

### Light version

....

### Dark version

....

## Modelina logo

### Light version
...

### Dark version
...
```

Reply:
Good idea! I will definitely implement this suggestion, thank you! 😄

Reply:
Okay so I realized why I did it like this... I wanted the TOC to link to each version of the logo, and if I have repetitive titles like `Light version` and `Dark version` for each one, I am wondering if it will allow me to link to those sections still? As the titles will not have unique IDs... Let me know if you know what I am talking about and if there is a solution for this if we were to change the structure to your suggestion.

Reply:
Yup, Markdown supports HTML so you can probably do something like this:

```markdown
### <a name="modelina-light" />Light version
```

or, if it doesn't work:

```markdown
<a name="modelina-light" />
### Light version
```

And then link to `#modelina-light` in the TOC.

> Now I'm not sure if it's `<a name=...` or `<a id=...`. Try both 😅

Reply:
Okay I believe I have resolved this. I didn't realize that markdown automatically adds incrementing numbers to the IDs that are duplicate headings. Let me know if this is okay?